build.sbt

```diff
@@ -222,26 +222,32 @@ lazy val `integration-test-local` = project
 
 lazy val docs = project
   .in(file("docs"))
-  .enablePlugins(AkkaParadoxPlugin)
+  .enablePlugins(AkkaParadoxPlugin, ParadoxSitePlugin, PublishRsyncPlugin)
   .disablePlugins(BintrayPlugin)
   .settings(
     name := "Akka Management",
     publish / skip := true,
     whitesourceIgnore := true,
+    makeSite := makeSite.dependsOn(LocalRootProject / ScalaUnidoc / doc).value,
+    previewPath := (Paradox / siteSubdirName).value,
     paradoxGroups := Map("Language" -> Seq("Java", "Scala")),
-    Compile / paradox := (Compile / paradox).dependsOn(LocalRootProject / ScalaUnidoc / doc).value,
+    Paradox / siteSubdirName := s"docs/akka-management/${if (isSnapshot.value) "snapshot" else version.value}",
```

Comment:
This is missing either an Preprocess sbt-site plugin, that will take scaladoc as input and fix up the java links, as we have in Alpakka. Or it needs API docs sbt-site plugin: https://www.scala-sbt.org/sbt-site/api-documentation.html